rules/establish-the-world/rule.md

```diff
@@ -0,0 +1,44 @@
+---
+type: rule
+title: Do you establish the world for your audience?
+uri: establish-the-world
+authors:
+  - title: Jonty Gardner
+  - url: https://ssw.com.au/people/jonty-gardner
+created: 2023-10-04T06:22:53.243Z
+guid: f1dac5ed-a422-4e19-bf72-f6c637ba0a23
+---
+## 🎥 Do you establish the world for your audience?
+
+When creating video content, it's crucial to start by setting up the world and the location where your content is taking place. This initial impression can significantly impact how your audience engages with your video. One effective way to achieve this is through the use of drone footage.
+
+Drones have become increasingly popular and offer the ability to capture breathtaking landscape shots and viewpoints that were once impossible to achieve a decade ago. Investing in drone footage not only adds a touch of professionalism but also elevates the overall production quality of your video.
+
+### Why Establishing the World Matters:
+
+- **Professionalism:** Incorporating drone footage signals a commitment to high-quality production, which can leave a lasting positive impression on your audience.
+
+- **Visual Impact:** Drone shots can showcase epic landscapes and unique perspectives, captivating viewers and making your content visually engaging.
+
+- **Spatial Awareness:** By showing the environment from above, drone footage helps viewers understand the geographical context of the video's location.
+
+### Examples:
+
+::: greybox {
+   [youtube: 'https://youtu.be/7jOJ2QlfGUg']
+} :::
+
+::: bad Figure: Bad Example
+The video starts abruptly without establishing the setting, leaving viewers confused about where the content is taking place.
+:::
+
+::: greybox {
+   [youtube: 'https://youtu.be/uFLnYo_z6Pw']
+} :::
+
+::: good Figure: Good Example
```

Comment:
@jontygardner 

1. Please fix to be like the below
2. Do the same for the bad example (attention to small case "example")

```
::: good 
Figure: Good example - The video begins (after the hook) with stunning drone footage of the scenic location, setting the stage for the content and immersing the audience in the world of the video.
:::
```